### ISSUE TITLE
EventSource exception for `PositionReceived` (null reference)

### ISSUE THREAD

**Issue by alexeyzimarev** [Eventuous/eventuous#133]
Seems to be a bug in the diagnostics

**Comment by alexeyzimarev** [comment]
```
EventSourceException while processing event "PositionReceived": System.NullReferenceException:Object reference not set to an instance of an object.
EventSourceException while processing event "Committing": System.NullReferenceException:Object reference not set to an instance of an object.
```

**Comment by alexeyzimarev** [comment]
Possibly related message:
```
RROR: Exception in Command Processing for EventSource eventuous-checkpoint: An instance of EventSource with Guid 6b810c27-1ead-5a19-94e9-8aa01409eca2 already exists.
```

**Comment by alexeyzimarev** [comment]
Fixed by dcdc9a0025a2ae2242c815e6156cef02f0771f16
